rclpy/rclpy/clock_type.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2024 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from enum import IntEnum
+
+from rclpy.impl.implementation_singleton import rclpy_implementation as _rclpy
+
+
+class ClockType(IntEnum):
```

Comment:
I lifted it out since time.py also needs ClockType to avoid circular dependencies. I could move ClockType to time.py and import from there if you want.